hw/ip/flash_ctrl/data/flash_ctrl_testplan.hjson

```diff
@@ -25,199 +25,325 @@
       tests: ["flash_ctrl_smoke"]
     }
     {
-      name: backpressure
+      name: smoke_hw_rd
       desc: '''
-            Sanity + randomly pick between back-pressured reads / writes and fifo status-polled
-            reads / writes via the controller.
+            Perform host direct read on the single page of one bank within Data partition.
+            '''
+      milestone: V1
+      tests: []
+    }
+    {
+      name: sw_op_pwrmgr
```

Comment:
How come this test is named "sw_op_pwrmgr"? I don't see what pwrmgr relates to here.

Reply:
It will be renamed to sw_op once this pull request is merged. I need to add test name for #9409 so then i am going to fix this.

Reply:
@matutem : I fixed this to sw_op since i needed to update pull request due to merged conflict. This is the only fix that i had beside resolving conflict with master branch.